common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -8,13 +8,16 @@
 # Import this just to export it
 from pymongo.errors import DuplicateKeyError  # pylint: disable=unused-import
 
-from contracts import check
+from contracts import check, new_contract
 from xmodule.exceptions import HeartbeatFailure
-from xmodule.modulestore.split_mongo import BlockKey
+from xmodule.modulestore.split_mongo import BlockKey, BlockData
 import datetime
 import pytz
 
 
+new_contract('BlockData', BlockData)
```

Comment:
This statement tells PyContracts: "When you see BlockData in a `contract` decorator or a `check` statement, I'm referring to the BlockData class." We've given the PyContracts author feedback that it'd be good to eliminate the need for this syntax, as it's redundant - and he's working on it.